### PR TITLE
Issue 1 create an entry - pending approval

### PR DESCRIPTION
**Pull request by junoulee**

### CODE REVIEW

**Comment by junoulee** [comment]
feature 1

File: index.html [hunk 1]
```diff
@@ -16,12 +16,69 @@
...
+     alt="Photo URL Image">
+              </div>
+            </div>
+          </div>
+
+          <div class="column-half">
+
+            <div class="form-box">
+              <div class="title-input">
+                <p>Title</p>
```
Comment (codingcodymiller) [inline index.html]: This should be a `label`. Make sure you configure it correctly, I recommend reviewing `html-forms`

File: index.html [hunk 2]
```diff
@@ -16,12 +16,69 @@
...
+                <input required class="input-text" name="title" type="text" size="15"/>
+                <p>Photo URL</p>
```
Comment (codingcodymiller) [inline index.html]: This should be a `label`. Make sure you configure it correctly, I recommend reviewing `html-forms`

File: index.html [hunk 3]
```diff
@@ -16,12 +16,69 @@
...
+          </div>
+        </div>
+
+        <div class="row">
+
+          <div class="column-full">
+            <div class="bottom-section">
+              <div class="form-box">
+                <div class="notes-section">
+                  <p>Notes</p>
```
Comment (codingcodymiller) [inline index.html]: This should be a `label`. Make sure you configure it correctly, I recommend reviewing `html-forms`

File: css/styles.css [hunk 4]
```diff
@@ -0,0 +1,108 @@
...
+  color: white;
+  margin-top: 0;
+  height: 125%;
+  font-family: sans-serif;
+  font-weight: 100;
+  align-items: center;
+  padding-left: 15px;
+  padding-bottom: 15px;
+  font-size: 1.2rem;
+}
```
Comment (codingcodymiller) [inline css/styles.css]: What is this class for? I don't see it used anywhere. You should delete it.

File: js/main.js [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+var placeHolder = document.querySelector('.placeholder');
+var $form = document.querySelector('.journal-form');
+
+function showPic(event) {
+  placeHolder.setAttribute('src', event.target.value);
+}
+
+photoPreview.addEventListener('input', showPic);
+
+data.nextEntryId = 0;
```
Comment (codingcodymiller) [inline js/main.js]: This should not be done.

File: js/main.js [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+
+function storeValues(event) {
+  event.preventDefault();
+  var titleValue = $form.elements.title.value;
+  var urlValue = $form.elements.url.value;
+  var notesValue = $form.elements.notes.value;
+  var fieldEntries = { title: titleValue, url: urlValue, notes: notesValue, entryId: data.nextEntryId };
+  data.entries.unshift(fieldEntries);
+
+  if (data.entries.length++) {
```
Comment (codingcodymiller) [inline js/main.js]: Why are you trying to increment the length of the array in a conditional?? 😨 

You can delete this conditional, just keep the code inside of it.

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

**Review by codingcodymiller — APPROVED** [review]

**Pull request merged**